2016-03-07 14:24:41+0300 [-] Log opened.
2016-03-07 14:24:41+0300 [-] PBServerFactory starting on 13333
2016-03-07 14:24:41+0300 [-] Starting factory <twisted.spread.pb.PBServerFactory instance at 0x048EBF80>
2016-03-07 14:24:41+0300 [-] Starting off the system
2016-03-07 14:25:14+0300 [-] Daily Thread initiating messenger
2016-03-07 14:25:14+0300 [-] 5 messages waiting to be sent
2016-03-07 14:25:35+0300 [-] Internet error
2016-03-07 14:25:35+0300 [-] False
2016-03-07 14:25:56+0300 [-] Internet error
2016-03-07 14:25:56+0300 [-] False
2016-03-07 14:26:17+0300 [-] Internet error
2016-03-07 14:26:17+0300 [-] False
2016-03-07 14:26:38+0300 [-] Internet error
2016-03-07 14:26:38+0300 [-] False
2016-03-07 14:26:41+0300 [-] [(u'+254704236788', u'Success', u'KES 1.0000', 'Blank message')]
2016-03-07 14:27:03+0300 [-] Hourly Thread initiating messenger
2016-03-07 14:27:03+0300 [-] 4 messages waiting to be sent
2016-03-07 14:27:04+0300 [-] [(u'+254704236788', u'Success', u'KES 1.0000', 'Blank')]
2016-03-07 14:27:05+0300 [-] [(u'+254704236788', u'Success', u'KES 1.0000', 'Blank')]
2016-03-07 14:27:07+0300 [-] [(u'+254704236788', u'Success', u'KES 1.0000', 'Blank')]
2016-03-07 14:27:08+0300 [-] [(u'+254704236788', u'Success', u'KES 1.0000', 'Blank')]
2016-03-07 14:28:50+0300 [Broker,0,127.0.0.1] [('+254704236788', ' Test message fom K-BIMA')]
2016-03-07 14:28:53+0300 [Broker,0,127.0.0.1] [(u'+254704236788', u'Success', u'KES 1.0000', ' Test message fom K-BIMA')]
2016-03-07 14:31:28+0300 [-] Received SIGINT, shutting down.
2016-03-07 14:31:28+0300 [twisted.spread.pb.PBServerFactory] (TCP Port 13333 Closed)
2016-03-07 14:31:28+0300 [twisted.spread.pb.PBServerFactory] Stopping factory <twisted.spread.pb.PBServerFactory instance at 0x048EBF80>
2016-03-07 14:31:28+0300 [-] Connection close failed
	Traceback (most recent call last):
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 295, in addCallbacks
	    self._runCallbacks()
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 577, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "d:\python27\lib\site-packages\twisted\internet\base.py", line 429, in _continueFiring
	    callable(*args, **kwargs)
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 393, in finalClose
	    self._close(conn)
	--- <exception caught here> ---
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 439, in _close
	    conn.close()
	sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread.The object was created in thread id 7648 and this is thread id 5616
	
2016-03-07 14:31:28+0300 [-] Connection close failed
	Traceback (most recent call last):
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 295, in addCallbacks
	    self._runCallbacks()
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 577, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "d:\python27\lib\site-packages\twisted\internet\base.py", line 429, in _continueFiring
	    callable(*args, **kwargs)
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 393, in finalClose
	    self._close(conn)
	--- <exception caught here> ---
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 439, in _close
	    conn.close()
	sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread.The object was created in thread id 1564 and this is thread id 5616
	
2016-03-07 14:31:28+0300 [-] Main loop terminated.
2016-03-07 14:31:33+0300 [-] Log opened.
2016-03-07 14:31:33+0300 [-] PBServerFactory starting on 13333
2016-03-07 14:31:33+0300 [-] Starting factory <twisted.spread.pb.PBServerFactory instance at 0x04B16F80>
2016-03-07 14:31:33+0300 [-] Starting off the system
2016-03-07 14:32:05+0300 [-] Daily Thread initiating messenger
2016-03-07 14:32:05+0300 [-] 0 messages waiting to be sent
2016-03-07 14:32:19+0300 [-] Unhandled error in Deferred:
2016-03-07 14:32:19+0300 [-] Unhandled Error
	Traceback (most recent call last):
	  File "runserver.py", line 565, in <module>
	    main()
	  File "runserver.py", line 561, in main
	    task.LoopingCall(manager.hourly_messenger).start(3600)
	  File "d:\python27\lib\site-packages\twisted\internet\task.py", line 173, in start
	    self()
	  File "d:\python27\lib\site-packages\twisted\internet\task.py", line 218, in __call__
	    d = defer.maybeDeferred(self.f, *self.a, **self.kw)
	--- <exception caught here> ---
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 139, in maybeDeferred
	    result = f(*args, **kw)
	  File "D:\Projects\CMS\server1.3\cron.py", line 318, in hourly_messenger
	    time.sleep(20)
	exceptions.KeyboardInterrupt: 
	
2016-03-07 14:32:22+0300 [-] Received SIGINT, shutting down.
2016-03-07 14:32:22+0300 [twisted.spread.pb.PBServerFactory] (TCP Port 13333 Closed)
2016-03-07 14:32:22+0300 [twisted.spread.pb.PBServerFactory] Stopping factory <twisted.spread.pb.PBServerFactory instance at 0x04B16F80>
2016-03-07 14:32:22+0300 [-] Main loop terminated.
2016-03-07 14:32:24+0300 [-] Log opened.
2016-03-07 14:32:24+0300 [-] PBServerFactory starting on 13333
2016-03-07 14:32:24+0300 [-] Starting factory <twisted.spread.pb.PBServerFactory instance at 0x029C5800>
2016-03-07 14:32:24+0300 [-] Starting off the system
2016-03-07 14:32:56+0300 [-] Daily Thread initiating messenger
2016-03-07 14:32:56+0300 [-] 0 messages waiting to be sent
2016-03-07 14:33:18+0300 [-] Hourly Thread initiating messenger
2016-03-07 14:33:18+0300 [-] 0 messages waiting to be sent
2016-03-07 14:33:21+0300 [Broker,0,127.0.0.1] [('+254704236788', ' Test message fom K-BIMA')]
2016-03-07 14:33:24+0300 [Broker,0,127.0.0.1] [(u'+254704236788', u'Success', u'KES 1.0000', ' Test message fom K-BIMA')]
2016-03-07 14:33:42+0300 [-] Received SIGINT, shutting down.
2016-03-07 14:33:42+0300 [twisted.spread.pb.PBServerFactory] (TCP Port 13333 Closed)
2016-03-07 14:33:42+0300 [twisted.spread.pb.PBServerFactory] Stopping factory <twisted.spread.pb.PBServerFactory instance at 0x029C5800>
2016-03-07 14:33:42+0300 [-] Connection close failed
	Traceback (most recent call last):
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 295, in addCallbacks
	    self._runCallbacks()
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 577, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "d:\python27\lib\site-packages\twisted\internet\base.py", line 429, in _continueFiring
	    callable(*args, **kwargs)
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 393, in finalClose
	    self._close(conn)
	--- <exception caught here> ---
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 439, in _close
	    conn.close()
	sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread.The object was created in thread id 8248 and this is thread id 712
	
2016-03-07 14:33:42+0300 [-] Connection close failed
	Traceback (most recent call last):
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 295, in addCallbacks
	    self._runCallbacks()
	  File "d:\python27\lib\site-packages\twisted\internet\defer.py", line 577, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "d:\python27\lib\site-packages\twisted\internet\base.py", line 429, in _continueFiring
	    callable(*args, **kwargs)
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 393, in finalClose
	    self._close(conn)
	--- <exception caught here> ---
	  File "d:\python27\lib\site-packages\twisted\enterprise\adbapi.py", line 439, in _close
	    conn.close()
	sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread.The object was created in thread id 4012 and this is thread id 712
	
2016-03-07 14:33:42+0300 [-] Main loop terminated.
2016-03-08 10:30:17+0300 [-] Log opened.
2016-03-08 10:30:17+0300 [-] PBServerFactory starting on 13333
2016-03-08 10:30:17+0300 [-] Starting factory <twisted.spread.pb.PBServerFactory instance at 0x02871800>
2016-03-08 10:30:17+0300 [-] Starting off the system
2016-03-08 10:30:49+0300 [-] Daily Thread initiating messenger
2016-03-08 10:30:49+0300 [-] this is the type extension
2016-03-08 10:30:49+0300 [-] extension has been disabled
2016-03-08 10:30:49+0300 [-] this is the type renewal
2016-03-08 10:30:49+0300 [-] renewal has been disabled
2016-03-08 10:30:49+0300 [-] this is the type clients
2016-03-08 10:30:49+0300 [-] clients has been disabled
2016-03-08 10:30:49+0300 [-] this is the type newinvoice
2016-03-08 10:30:49+0300 [-] newinvoice has been disabled
2016-03-08 10:30:49+0300 [-] this is the type birthday
2016-03-08 10:30:49+0300 [-] birthday has been disabled
2016-03-08 10:30:49+0300 [-] this is the type cheque
2016-03-08 10:30:49+0300 [-] cheque has been disabled
2016-03-08 10:30:49+0300 [-] this is the type balance
2016-03-08 10:30:49+0300 [-] balance has been disabled
2016-03-08 10:30:49+0300 [-] 0 messages waiting to be sent
2016-03-08 10:31:11+0300 [-] Hourly Thread initiating messenger
2016-03-08 10:31:11+0300 [-] 0 messages waiting to be sent
2016-03-08 10:31:42+0300 [Broker,0,127.0.0.1] admin asking for authentification
2016-03-08 10:31:45+0300 [Broker,1,127.0.0.1] Server responding to request with 2592 renewals
2016-03-08 10:31:45+0300 [Broker,1,127.0.0.1] Server responding to request with 670 balances
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 670 clients
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 1999 expiries
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 0 cheques
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 0 bank
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 0 payee
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 0 cheque types
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 13 users
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with company information
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 0 names
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 4 group clients
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to request with 7 messages
2016-03-08 10:31:46+0300 [Broker,1,127.0.0.1] Server responding to outbox request
